decisions/0004-issue-triage.md

```diff
@@ -0,0 +1,50 @@
+# 4. Issue Triage for OSCAL Core Projects
+
+Date: 2023/03/03
+
+## Status
+
+Feedback Requested
```

Comment:
You know, for when you we actually do want to merge with the correct status next time.

```suggestion
Approved
```